internal/provider/external/installConfig.go

```diff
@@ -15,6 +16,9 @@ func (p externalProvider) AddPlatformToInstallConfig(cfg *installcfg.InstallerCo
 		None: &installcfg.PlatformNone{},
 	}
 
+	// Enable feature set "TechPreviewNoUpgrade" in order to access External platform feature
```

Comment:
@filanov This code does not exist in the master code, see: https://github.com/openshift/assisted-service/blob/master/internal/provider/external/installConfig.go